infra/prover-cluster/index.ts

```diff
@@ -27,7 +27,7 @@ const environment: string = config.get("environment") || "custom";
 const privateKey: pulumi.Output<string> = config.requireSecret("privateKey");
 const ethRpcUrl: pulumi.Output<string> = config.requireSecret("ethRpcUrl");
 const managerInstanceType: string = config.require("managerInstanceType");
-const orderStreamUrl: string = config.require("orderStreamUrl");
+const orderStreamUrl: pulumi.Output<string> = config.requireSecret("orderStreamUrl"); // Use secret to avoid exposing staging urls
```

Comment:
I'm maybe just overlooking something, but won't this break release as those are in plaintext? Or does it allow either?